tests/test_repro.py

```diff
@@ -1021,3 +1095,96 @@ def test(self):
             ret = main(['repro', '--all-pipelines'])
             self.assertEqual(ret, 0)
             self.assertEqual(mock_reproduce.call_count, 4)
+
+
+class TestReproAlreadyCached(TestDvc):
+    def test_import(self):
```

Comment:
This is no longer needed, since you have this test in TestExternalBase, which also tests local deps/outs.